server/service/orbit_client.go

```diff
@@ -368,6 +369,14 @@ func OrbitRetryInterval() time.Duration {
 	return constant.OrbitEnrollRetrySleep
 }
 
+func OrbitRetryBackoff() bool {
+	backoff := os.Getenv("FLEETD_ENROLL_BACKOFF")
```

Comment:
We usually define configuration in orbit as a new flag (which also allows to configure it as env variable).
(And for consistency, we use the `ORBIT_` prefix.)

Example:
https://github.com/fleetdm/fleet/blob/653d4be7f9bf9a9b641c635ba73cfc6ef5349f08/orbit/cmd/orbit/orbit.go#L200-L204